src/psychrometrics/t_mrt.js

```diff
@@ -0,0 +1,162 @@
+import { valid_range } from "../utilities/utilities";
+
+const g = 9.81;
+const c_to_k = 273.15;
+const cp_air = 1004;
+
+/**
+ * Converts globe temperature reading into mean radiant temperature in accordance with either the Mixed Convection developed by Teitelbaum E. et al. (2022) or the ISO 7726:1998 Standard [5]_.
+ * @param {Number | Number[]} tg - globe temperature, [°C]
+ * @param {Number | Number[]} tdb - air temperature, [°C]
+ * @param {Number | Number[]} v - air speed, [m/s]
+ * @param {Number | Number[]} d - diameter of the globe, [m] default 0.15 m
+ * @param {Number | Number[]} emissivity - emissivity of the globe temperature sensor, default 0.95
+* @param {String} standard - {"Mixed Convection", "ISO"} either choose between the Mixed Convection and ISO formulations.
```

Comment:
change `String` to `"Mixed Convection" | "ISO"` so our users get intellisense/autocomplete with the options